newsplease/crawler/spiders/recursive_sitemap_crawler.py

```diff
@@ -28,16 +28,16 @@ def __init__(self, helper, url, config, ignore_regex, *args, **kwargs):
         self.helper = helper
 
         self.ignore_regex = ignore_regex
-        self.ignore_file_extensions = self.config.section(
-            'Crawler')['ignore_file_extensions']
+        self.ignore_file_extensions = self.config.section("Crawler")[
+            "ignore_file_extensions"
+        ]
 
         self.original_url = url
 
-        self.allowed_domains = [self.helper.url_extractor
-                                    .get_allowed_domain(url)]
-        self.sitemap_urls = [self.helper.url_extractor.get_sitemap_url(
-            url, config.section('Crawler')['sitemap_allow_subdomains'])]
-
+        self.allowed_domains = [self.helper.url_extractor.get_allowed_domain(url)]
+        self.sitemap_urls = self.helper.url_extractor.get_sitemap_urls(
```

Comment:
Here we now retrieve a list of strings containing either robots.txt data (like before) or available sitemaps